src/ansys/mechanical/core/__init__.py

```diff
@@ -4,6 +4,15 @@
 
 import appdirs
 
+# Setup data directory
+USER_DATA_PATH = appdirs.user_data_dir(appname="ansys_mechanical_core", appauthor="Ansys")
+if not os.path.exists(USER_DATA_PATH):
+    os.makedirs(USER_DATA_PATH)
+
+EXAMPLES_PATH = os.path.join(USER_DATA_PATH, "examples")
+if not os.path.exists(EXAMPLES_PATH):
+    os.makedirs(EXAMPLES_PATH)
+
```

Comment:
Why removing the try/except? Pinging @koubaa here

Reply:
Good question. The main reason is I don't see a reason for it (German didn't see a reason for it in pyMAPDL either). I am assuming that folder creation is important for this module (e.g. settings_dir) and/or tests, and so hiding exceptions here and failing silently does not look like a good idea to me (I may be wrong obviously). 

If this application does not need the folders created (and failing to create folders doesn't matter, so that we could hide potential failure to create/check for folders with this try-except block), then why are we even creating these folders here to begin with, every time the module is imported?

No strong opinions either way, just my thoughts. Let me know and I can easily add the try-except block back if preferred.

Reply:
Okay, sounds good to me then! This was probably "taken" from PyMAPDL nonetheless. Let's leave it without the try/catch =)